src/coreclr/tools/Common/Compiler/HardwareIntrinsicHelpers.cs

```diff
@@ -16,7 +16,7 @@ public static partial class HardwareIntrinsicHelpers
         /// </summary>
         public static bool IsHardwareIntrinsic(MethodDesc method)
         {
-            return !string.IsNullOrEmpty(InstructionSetSupport.GetHardwareIntrinsicId(method.Context.Target.Architecture, method.OwningType));
+            return !string.IsNullOrEmpty(InstructionSetSupport.GetHardwareIntrinsicId(method.Context.Target.Architecture, method.OwningType, out bool _));
```

Comment:
nit: `out _` for C# discards (typed _ are real variables and subjected to unused variable warnings)